### DISCUSSION TITLE
Mac OS

### DISCUSSION THREAD

**Discussion by drinehydrogen** [hydrogen-music/hydrogen#1973] [Q&A]
Bonjour, j'ai un macOs de 2015, puis je trouver une version compatible d'hydrogen?
Celle que je trouve sur le site semble compatible uniquement avec macX
Merci de votre réponse

**Edit:** translated by @trebmuh :
> Hello, I have a 2015 macOs, can I find a compatible version of hydrogen?
> The one I found on the site seems to be compatible only with macX.
> Thank you for your reply

**Comment by theGreatWhiteShark** [comment]
Hey @drinehydrogen ,

Could you rephrase your question in English (or German, Spanish, or Russian)?

**Reply by drinehydrogen** [reply]
it seems i found a solution. I will come back to you if it doesn't work. Thanks

**Comment by cme** [comment]
The build works more or less out of the box with MacPorts packages on older Mac OS X versions, at least as far back as High Sierra. I have a build script that'll do the necessary setup which perhaps I should add.

**Reply by cme** [reply]
Actually, I've just found that macports already have Hydrogen as a port! So if you install macports ( https://www.macports.org/install.php ) then do
```sh
sudo port install hydrogen
```
you should get the latest Hydrogen in /Applications/MacPorts!

**Reply by theGreatWhiteShark** [reply]
Alternatively you could also try to [build Hydrogen from source](https://github.com/hydrogen-music/hydrogen/wiki/Building-Hydrogen-from-Source-(macOS)).
